### ISSUE TITLE
ExcelRange.Copy does not copy and deletes conditional formatting

### ISSUE THREAD

**Issue by keithclarkj** [EPPlusSoftware/EPPlus#1272]
When a cell containing a conditional formatting rule is copied to an `ExcelRange`, the conditional formatting from the original document is deleted and not copied.

As well as this, when the packaged is newed up again, the `ExcelConditionalFormattingRules` still contains the rule, even though it does not exist in the final file.

v.7.0.8

Test file (Sheet2 contains 1 `GreaterThan` rule on cell A1):
[ConditionalFormatting.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/14088678/ConditionalFormatting.xlsx)

```
[TestMethod]
[DeploymentItem("Resources\\ConditionalFormatting.xlsx")]
public void CopyCellWithConditionalFormatting()
{
    using (var pck = new ExcelPackage("ConditionalFormatting.xlsx"))
    {
        var sheet = pck.Workbook.Worksheets["Sheet2"];

        var cellToCopy = sheet.Cells["A1"];
        var newCellRange = sheet.Cells["B1"];

        cellToCopy.Copy(newCellRange);

        var conditionalFormattingCollection = sheet.ConditionalFormatting; // still contains 1 rule even though it is gone in file

        pck.SaveAs([save path]); 
        // After save conditional formatting is not copied and is deleted from the original cell

        using (var pck = new ExcelPackage([save path]))
        {
            var conditionalFormattingRules = pck.Workbook.Worksheets["Sheet2"].ConditionalFormatting.Count;
            Assert.AreEqual(1, conditionalFormattingRules); // this returns a success, however when opening the file the rule is gone
        }
    }
}
```

**Comment by OssianEPPlus** [comment]
This is a bug, looks like we had a comma in our addresses when writing to file instead of space-seperated. Which resulted in the data being written and read just fine by epplus but Excel did not understand what address to apply the info to.

A fix is underway.

**Comment by keithclarkj** [comment]
@OssianEPPlus I see, thank you!

**Comment by OssianEPPlus** [comment]
A fix for this should now be available via [Our NuGet Develop Feed](https://github.com/EPPlusSoftware/EPPlus/wiki/Using-our-develop-Nuget-feed). And is slated for next release.

**Comment by OssianEPPlus** [comment]
This fix has been released in epplus 7.0.9
